examples/local/launch_cloud_onprem.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import os
+import subprocess
+import tempfile
+import textwrap
+import uuid
+import yaml
+
+from click import testing as cli_testing
+
+from sky import cli
+from sky import global_user_state
+from sky.backends import onprem_utils
+from sky.utils import common_utils
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--local-cluster-name',
```

Comment:
If this is a required arg, make it a positional arg? or add a short hand `-n`

Reply:
Done, thanks!